Cargo.toml

```diff
@@ -59,12 +62,13 @@ pretty_assertions = "0.4"
 walkdir = "2.0"
 
 [features]
-default = ["output", "watch", "serve"]
+default = ["output", "watch", "serve", "search"]
 debug = []
 output = []
 regenerate-css = []
 watch = ["notify", "time", "crossbeam"]
 serve = ["iron", "staticfile", "ws"]
+search = ["elasticlunr-rs"]
```

Comment:
Forgot that I added this feature, it doesn't currently do anything. If desired, I can figure out how to extract the search functionality from hbs_renderer so it can be disabled.